### PR TITLE
feat(@vtmn/svelte): add `VtmnRadioButton` component

### PR DESCRIPTION
**Pull request by lpoupet**
## Changes description
Add RadioButton component to vitamine svelte library.

## Other information
image

### CODE REVIEW

File: packages/showcases/svelte/stories/components/selection-controls/VtmnRadioButton/VtmnRadioButton.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      name="favorite-choice"
+      label="Ski"
+      checked={true}
+    />
+    <VtmnRadioButton id="your-choice-2" name="favorite-choice" label="MTB" />
+    <VtmnRadioButton
+      id="your-choice-3"
+      name="favorite-choice"
+      label="Badminton"
+      disabled={true}
```
Comment (Tlahey) [inline packages/showcases/svelte/stories/components/selection-controls/VtmnRadioButton/VtmnRadioButton.stories.svelte:39]: Same here ^^

File: packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/VtmnRadioButton.svelte [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+<div>
+  <input
+    class="vtmn-radio-button"
+    type="radio"
+    {id}
+    {name}
+    {checked}
+    {disabled}
+  />
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/VtmnRadioButton.svelte:46]: Please add `$$restProps` :) 

```javascript
<input
    class="vtmn-radio-button"
    type="radio"
    {id}
    {name}
    {checked}
    {disabled}
    {...$$restProps}
  />
```

File: packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/test/vtmnRadioButton.spec.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    const { getByLabelText, getByText } = render(VtmnRadioButton, params);
+    expect(getByLabelText(label)).toBeInTheDocument();
+    expect(getByText(label)).toBeInTheDocument();
+  });
+
+  test('Radio Button Should be selectable', () => {
+    const { getByLabelText } = render(VtmnRadioButton, params);
+    const radioButton = getByLabelText(label);
+    expect(radioButton).not.toBeChecked();
+    fireEvent.click(radioButton);
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/test/vtmnRadioButton.spec.js:22]: `await fireEvent.click(...)` and the function should be `async`

**Comment by Tlahey** [comment]
Great PR good job @lpoupet !

File: packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/test/vtmnRadioButton.spec.js (hunk 3 above)
Comment (lpoupet) [reply packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/test/vtmnRadioButton.spec.js:22]: I'm not sure, on [doc](https://testing-library.com/docs/dom-testing-library/api-events/) they don't use async func for this.

File: packages/showcases/svelte/stories/components/selection-controls/VtmnRadioButton/VtmnRadioButton.stories.svelte [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  </VtmnRadioButton>
+</Template>
+
+<Story
+  name="Overview"
+  args={{
+    id: 'id',
+    name: 'name',
+    label: 'label',
+  }}
```
[the remark this answers is not in the document]
Comment (lpoupet) [reply packages/showcases/svelte/stories/components/selection-controls/VtmnRadioButton/VtmnRadioButton.stories.svelte]: I changed everything. To have a real example (without `undefined` label)

**Comment by lauthieb** [comment]
@lpoupet sorry there's a problem with your tests suite. For the moment I convert your PR to draft. Feel free to click on `Ready for review` when it is ok for you. :)

File: packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/test/vtmnRadioButton.spec.js (hunk 3 above)
Comment (Tlahey) [reply packages/sources/svelte/src/components/selection-controls/VtmnRadioButton/test/vtmnRadioButton.spec.js:23]: This one is the example for svelte ->. https://testing-library.com/docs/svelte-testing-library/example

File: packages/showcases/svelte/stories/components/selection-controls/VtmnRadioButton/VtmnRadioButton.stories.svelte (hunk 4 above)
[the remark this answers is not in the document]
Comment (Tlahey) [reply packages/showcases/svelte/stories/components/selection-controls/VtmnRadioButton/VtmnRadioButton.stories.svelte]: You have to define `defaultValue` on the `argTypes`

File: packages/sources/svelte/package.json [hunk 5]
```diff
@@ -39,6 +39,7 @@
     "@vtmn/css-card": "^0.7.0",
     "@vtmn/css-checkbox": "^0.6.16",
     "@vtmn/css-chip": "^0.6.13",
+    "@vtmn/css-radio-button": "0.6.15",
```
Comment (GaspardMathon) [inline packages/sources/svelte/package.json:42]: Can you keep an alphabetical order please :)

**Review by GaspardMathon — CHANGES_REQUESTED** [review]

**Review by lauthieb — APPROVED** [review]

**Comment by lauthieb** [comment]
Thanks a lot @lpoupet, well done :tada:

**Pull request merged**